src/index.js

```diff
@@ -179,6 +181,23 @@ class DAPIClient {
    */
   getHistoricBlockchainDataSyncStatus() { return this.makeRequestToRandomDAPINode('getHistoricBlockchainDataSyncStatus', {}); }
 
+  /**
+   * Retrieve user's last state transition hash
+   * or regTxId in case no state transitions present
+   * @param {string} userId
+   * @returns {Promise<!LastUserStateTransitionHashResponse>}
+   */
+  async getLastUserStateTransitionHash(userId) {
+    const request = new LastUserStateTransitionHashRequest();
+    request.setUserId(userId);
+
+    const nodeToConnect = await this.MNDiscovery.getRandomMasternode();
+
+    const client = new CorePromiseClient(`${nodeToConnect.getIp()}:${this.getGrpcPort()}`);
+
+    return client.getLastUserStateTransitionHash(request);
```

Comment:
I think it will be easier for developers if you obtain a hash from the response object and return just string.